Ryujinx.HLE/HOS/Services/Prepo/IPrepoService.cs

```diff
@@ -93,103 +92,12 @@ public string ReadReportBuffer(byte[] buffer, string room, UInt128 userId)
 
             sb.AppendLine($" Room: {room}");
 
-            try
+            foreach (KeyValuePair<object, object> field in MessagePackSerializer.Deserialize<dynamic>(buffer))
```

Comment:
```suggestion
            var payload = MessagePackSerializer.Deserialize<IDictionary<string, string>>(buffer);
            
            foreach (var field in payload)
```

It's better to statically type the deserialization type when using MessagePack. In this case we wish to indicate that we expect string data in both the key and value. This will help catch issues earlier when the guest sends through data that isn't string key/value pairs.